chia/wallet/wallet_coin_store.py

```diff
@@ -18,9 +17,10 @@ class WalletCoinStore:
     """
 
     db_connection: aiosqlite.Connection
+    # coin_record_cache keeps ALL coin records in memory. [record_name: record]
```

Comment:
This is fine for wallet, but not great for full node. I ran some simulations and at height 500k it slowed down significantly with all coins in memory, FYI

Reply:
is it important to keep these in memory? I would expect sqlite to have its own cache, and that lookups for the same coins would be quick. In fact, I would trust sqlite's cache and the page cache more than our python collections.

Reply:
@mariano54 Yes, this is only for wallet. 
@arvidn Yes, performance. SQlite memory cache is limited in configurability, max size of memory is not per table configurable. Header blocks would dominate the coin records.

Reply:
Also the coin set (even for whales) is not that big.